package.json

```diff
@@ -29,6 +29,7 @@
     "poloniex.js": "0.0.6",
     "semver": "2.2.1",
     "sqlite3": "3.1.3",
+    "websocket": "latest",
```

Comment:
unfortunately this does not work on windows. I'll try to find an alternative which is more cross platform.